docs/getting-started/integration.md

```diff
@@ -84,3 +84,18 @@ require(['moment'], function() {
     });
 });
 ```
+
+## Content-Security-Policy
+
+By default, Chart.js injects CSS directly into the DOM. For webpages secured using [Content-Security-Policy (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy), this requires to allow `style-src 'unsafe-inline'`. For stricter CSP environments, where only `style-src 'self'` is allowed, the following CSS file needs to be manually added to your webpage:
+
+```html
+<link rel="stylesheet" type="text/css" href="path/to/chartjs/dist/Chart.min.css">
+```
+
+And the style injection must be turned off **before creating the first chart**:
```

Comment:
I might say "And enabling loading of external stylesheets must be turned on" rather than "And the style injection must be turned off" so that the language matches the option name a little more closely

Reply:
I prefer to describe what the option really does, which is to prevent the style injection. Even if the option name `useExternalStylesheet` may be good enough, I'm not fully happy with it because it doesn't reflect exactly what's behind that feature. Actually, it could be a bit confusing because users could think that the lib will "automatically" use an external file (e.g. by injecting a style element with `href="path/to/Chart.min.css"` or whatever provided by the user).

Reply:
I renamed the option to `disableCSSInjection`